tutorial-week-3.md

```diff
@@ -0,0 +1,7 @@
+
+I would like to do my tutorial about HTML. At the same time that I am learning I can write so that way I will review 
+whatever I learn new everyday this week.
+Today was basics like elements in HTML. 
+How to start:
+<!DOCTYPE html> were we inform the the browser that we are using the last version of html and the begining of the document.
+<head> will be the next element and it identifies the top of the document including the metadata and it may include a document title.
```

Comment:
Good idea. You could do a tutorial about how to start an HTML page (maybe an `index.html` page?).